selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/WorkspaceDetailsMachines.java

```diff
@@ -193,8 +194,29 @@ public void waitRamAmount(String machineName, String ramAmount) {
         driver -> getRamAmount(machineName).equals(ramAmount));
   }
 
+  private void clickOnRamField(String machineName) {
+    seleniumWebDriverHelper.moveCursorToAndClick(getRamAmountTextFieldLocator(machineName));
+  }
+
+  private void moveCursorToEditButton(String machineName) {
+    seleniumWebDriverHelper.moveCursorTo(getEditButtonLocator(machineName));
+  }
+
   public void typeRamAmount(String machineName, String ramAmount) {
-    seleniumWebDriverHelper.setValue(getRamAmountTextFieldLocator(machineName), ramAmount);
+    moveCursorToEditButton(machineName);
+    clickOnRamField(machineName);
+    seleniumWebDriverHelper.setValue(getRamAmountTextFieldLocator(machineName), "");
+
+    if (ramAmount.isEmpty()) {
```

Comment:
It makes sense to check it at start of method.

Reply:
Yes, for a case when specified empty string